README.md

```diff
@@ -8,10 +8,14 @@ Upstream documentation can be found at https://github.com/GoogleCloudPlatform/sp
 ## Usage
 
 The Spark Operator may be deployed using the Juju command line as follows
+
 ```bash
-juju deploy spark-k8s
+juju deploy spark-k8s spark
```

Comment:
Please let me know if this is also affecting deployments that are not necessarily the upstream examples, if not, I think we can leave the Usage section add an Examples one and add specific instructions for running the examples. This seems very specific.

## Usage
```bash
juju deploy spark-k8s
```

## Examples
Running upstream examples (add link to upstream examples)
1. Deploy `spark-k8s` as `spark` due to the upstream examples requiring \<add longer explanation\>
`juju deploy spark-k8s spark`
2. Run examples \<add instructions to run examples\>
` commands to run examples`